### ISSUE TITLE
ERROR IN Protocol1_8TO1_9 IN REMAP OF BLOCK_ENTITY_DATA (0x09)

### ISSUE THREAD

**Issue by skbeh** [ViaVersion/ViaRewind#429]
### `/viaversion dump` Output

https://dump.viaversion.com/668a1a0f3a0b60e19ea24cec33d75346da03a0b17518b14ea044067eb4234aac

### Console Error

```
[22:33:35] [Netty Epoll Server IO #1/WARN]: [ViaVersion] ERROR IN Protocol1_8TO1_9 IN REMAP OF BLOCK_ENTITY_DATA (0x09)
[22:33:35] [Netty Epoll Server IO #1/WARN]: io.netty.handler.codec.EncoderException: com.viaversion.viaversion.exception.InformativeException: Please report this on the Via support Discord or open an issue on the relevant GitHub repository
[22:33:35] [Netty Epoll Server IO #1/WARN]: Source 0: com.viaversion.viaversion.api.protocol.remapper.PacketHandlers$1 (Anonymous)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:107)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:113)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:120)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:120)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.DefaultChannelPipeline.write(DefaultChannelPipeline.java:1015)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannel.write(AbstractChannel.java:301)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ProtocolLib.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelProxy.write(NettyChannelProxy.java:210)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ProtocolLib.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelProxy.write(NettyChannelProxy.java:199)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at net.minecraft.network.NetworkManager.doSendPacket(NetworkManager.java:459)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at net.minecraft.network.NetworkManager.lambda$sendPacket$10(NetworkManager.java:428)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ProtocolLib.jar//com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.lambda$proxyRunnable$2(NettyEventLoopProxy.java:48)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:406)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at java.base/java.lang.Thread.run(Thread.java:833)
[22:33:35] [Netty Epoll Server IO #1/WARN]: Caused by: com.viaversion.viaversion.exception.InformativeException: Please report this on the Via support Discord or open an issue on the relevant GitHub repository
[22:33:35] [Netty Epoll Server IO #1/WARN]: Source 0: com.viaversion.viaversion.api.protocol.remapper.PacketHandlers$1 (Anonymous)
[22:33:35] [Netty Epoll Server IO #1/WARN]: Caused by: java.lang.NullPointerException: Cannot invoke "com.viaversion.viaversion.libs.opennbt.tag.builtin.Tag.getValue()" because the return value of "com.viaversion.viaversion.libs.opennbt.tag.builtin.CompoundTag.get(String)" is null
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaRewind.jar//de.gerrygames.viarewind.protocol.protocol1_8to1_9.packets.WorldPackets$1.lambda$registerMap$0(WorldPackets.java:53)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.api.protocol.remapper.PacketHandlers.handle(PacketHandlers.java:158)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.api.protocol.AbstractProtocol.transform(AbstractProtocol.java:341)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.protocol.packet.PacketWrapperImpl.apply(PacketWrapperImpl.java:389)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.protocol.packet.PacketWrapperImpl.apply(PacketWrapperImpl.java:377)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.protocol.packet.PacketWrapperImpl.apply(PacketWrapperImpl.java:44)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.protocol.ProtocolPipelineImpl.transform(ProtocolPipelineImpl.java:122)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.connection.UserConnectionImpl.transform(UserConnectionImpl.java:312)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.connection.UserConnectionImpl.transformClientbound(UserConnectionImpl.java:291)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.bukkit.handlers.BukkitEncodeHandler.encode(BukkitEncodeHandler.java:57)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ViaVersion.jar//com.viaversion.viaversion.bukkit.handlers.BukkitEncodeHandler.encode(BukkitEncodeHandler.java:35)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:90)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:113)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:120)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:120)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:881)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:863)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:968)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:856)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.DefaultChannelPipeline.write(DefaultChannelPipeline.java:1015)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.AbstractChannel.write(AbstractChannel.java:301)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ProtocolLib.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelProxy.write(NettyChannelProxy.java:210)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ProtocolLib.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelProxy.write(NettyChannelProxy.java:199)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at net.minecraft.network.NetworkManager.doSendPacket(NetworkManager.java:459)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at net.minecraft.network.NetworkManager.lambda$sendPacket$10(NetworkManager.java:428)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at ProtocolLib.jar//com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.lambda$proxyRunnable$2(NettyEventLoopProxy.java:48)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:406)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[22:33:35] [Netty Epoll Server IO #1/WARN]:     at java.base/java.lang.Thread.run(Thread.java:833)
```

### Bug Description

The player is disconnected when logging.

### Steps to Reproduce

Load the following world and connect to the server.
https://github.com/skbeh/misc/releases/download/tag/world.tar.gz

### Expected Behavior

No console errors.

### Checklist

- [X] Via plugins are only running on **EITHER** the backend servers (e.g. Paper) **OR** the proxy (e.g. BungeeCord), **not on both**.
- [X] I have included a ViaVersion dump.
- [X] If applicable, I have included a paste (**not a screenshot**) of the error.
- [X] I have tried the latest build(s) from https://ci.viaversion.com/ and the issue still persists.
- [X] I have looked at https://www.spigotmc.org/resources/viarewind-legacy-support.52924/ and it does not offer a fix for my problem

**Comment by Barvalg** [comment]
![Platform: git--Purpur--1927%20%28MC%3A%201.19.3%29](https://img.shields.io/badge/Platform-git--Purpur--1927%20%28MC%3A%201.19.3%29-3498db?style=flat-square) </br>![ViaVersion (4.6.0-1.19.4-rc2-SNAPSHOT): Even with dev](https://img.shields.io/badge/ViaVersion%20(4.6.0--1.19.4--rc2--SNAPSHOT)-Even%20with%20dev-brightgreen?style=flat-square) </br>![ViaBackwards(4.6.0-1.19.4-pre4-SNAPSHOT): Even with dev](https://img.shields.io/badge/ViaBackwards%20(4.6.0--1.19.4--pre4--SNAPSHOT)-Even%20with%20dev-brightgreen?style=flat-square) </br>![ViaRewind(2.0.3-SNAPSHOT): Even with master](https://img.shields.io/badge/ViaRewind%20(2.0.3--SNAPSHOT)-Even%20with%20master-brightgreen?style=flat-square) </br>

**Comment by FlorianMichael** [comment]
Fixed in 3.0.0-SNAPSHOT
